docker/backend/Dockerfile

```diff
@@ -104,15 +104,6 @@ COPY ./keyman/ keyman/
 
 RUN python3 -m venv /opt/venv
 
-RUN set -x \
```

Comment:
I recognize this is duplicative @EngCaioFonseca ... The trick with this is sometimes Docker's behavior is inconsistent with regards especially to making the wheel library available, which is necessary for some of the container build steps.